html2markdown.py

```diff
@@ -213,8 +213,8 @@ def _markdownify(tag, _listType=None, _blockQuote=False, _listIndex=1):
 		tag.string = '\n---\n'
 		tag.unwrap()
 	elif tag.name == 'pre':
-		tag.insert_before('\n\n')
-		tag.insert_after('\n\n')
+		tag.insert_before('\n\n```\n')
```

Comment:
This will cause any `<pre>` element to be surrounded with a code fence, not just `<pre><code>`.

Reply:
For safety and clarity that's not bad, but makes it harder to convert back to html.

Maybe for `<pre>` we only indent as you already did, but for `<code>` we backtick explicitly? Then the process to reverse back to html has a method it can follow.